packages/salesforcedx-vscode-core/test/jest/commands/templates/apexGenerateUnitTestClass.test.ts

```diff
@@ -97,9 +99,10 @@ describe('apexGenerateUnitTestClass Unit Tests.', () => {
   it('Should used the passed parameters if provided.', async () => {
     const fileName = 'testFileName';
     const outputDir = 'testOutputDir';
-    await apexGenerateUnitTestClass(fileName, outputDir);
-    expect(selectFileNameMocked).not.toHaveBeenCalled();
-    expect(selectOutputDirMocked).not.toHaveBeenCalled();
+    await apexGenerateUnitTestClass(outputDir, fileName);
```

Comment:
Changed because the test's now reset the module cache. This causes the various gatherers to still be initialized, not necessarily used but still initiated.

Production code is still doing the same thing, this is only a test behavior that better matches reality.